packages/manager/apps/dedicated/client/app/app.module.js

```diff
@@ -298,6 +298,7 @@ export default (containerEl, environment) => {
     .run(
       /* @ngInject */ (
         $location,
+        $log,
```

Comment:
Shouldn't we configure `$logProvider` in order to have logs in the console only if we are in `development` or in `production` with a debug flag activated (in localStorage, url param, ...)?

If we want to debug in prod, we can set for example a localStorage item:
```
localStorage.setItem('MANAGER_DEBUG_FLAG', true);
```
And configure `$log` like this:
```
$logProvider.debugEnabled(localstorage.getItem('MANAGER_DEBUG_FLAG'));
```
Then when we refresh the page, we have logs from `$log` in the console. This can also be useful for other touchy code and we will be able to let some `$log` in our code in order to debug.

What do you think?

Reply:
Hi @Jisay!

I have taken into account your feedback. Please let me know if changes looks good to you.

Thank you!

Reply:
@Jisay Could you please take a look and tell me if changes looks good for you?
Thank you!